src/features/views/components/ViewColumnDialog/PersonTagConfig.tsx

```diff
@@ -0,0 +1,45 @@
+import { FormControl } from '@mui/material';
+import TagManager from 'features/tags/components/TagManager';
+import { useState } from 'react';
+import { ZetkinTag } from 'utils/types/zetkin';
+
+import { COLUMN_TYPE, SelectedViewColumn } from '../types';
```

Comment:
We usually put imports in two groups: 

* First all the imports from external libraries (in this case MUI and react)
* Then imports from the modules within the project itself (in this case `TagManager`, `ZetkinTag` and `COLUMN_TYPE` etc)

To be consistent with that pattern, lines 2 and 4 should be moved down and be grouped together with line 6 (just above it).

Reply:
Got it!